src/TaskManager/TaskManager.cs

```diff
@@ -198,9 +203,16 @@ private async Task HandleDispatchTask(JsonMessage<TaskDispatchEvent> message)
 
             try
             {
-                PopulateTemporaryStorageCredentials(message.Body.Inputs.ToArray());
-                PopulateTemporaryStorageCredentials(message.Body.IntermediateStorage);
-                PopulateTemporaryStorageCredentials(message.Body.Outputs.ToArray());
+                if (message.Body.TaskPluginType == "Argo")
```

Comment:
Is there any reason to treat it differently for Argo?

Reply:
@JJFinnerty might be able to explain a bit better but I'm told Argo cant handle temporary credentials to put it simply.

Reply:
That's correct; Argo can't handle session-based temporary credentials.  Also, I think there's a `string` resource for `"Argo"`.

Reply:
I couldn't find a usable string resource initially saw other magic strings else where... 
(there's the Strings.cs inside TaskManager.Argo for application Id there but that is out of scope of the TaskManager.)

So just made a commit to make changes in the code that introduces a PluginStrings.cs put it in there for names of plugins.

Reply:
Yes, sorry for the confusion.  It's defined in `TaskManagerConfiguration.PluginAssemblyMappings` as a `Dictionary<,>`.  So the name comes from `appsettings.json`:

```
"taskManager": {
      "concurrency": 10,
      "taskTimeoutMinutes": 60,
      "plug-ins": {
        "argo": "Monai.Deploy.WorkflowManager.TaskManager.Argo.ArgoPlugin, Monai.Deploy.WorkflowManager.TaskManager.Argo"
      }
    }
```

Given that users may be entering anything they like, e.g. `Argo`, `ARGO`, `argo`, we may want to do a case-insensitive comparison.